### PR TITLE
Wizard recipe: oneTBB-v1.0.0

### PR DESCRIPTION
**Pull request by pitsianis**
This pull request contains a new build recipe I built using the BinaryBuilder.jl wizard:

* Package name: oneTBB
* Version: v1.0.0

@staticfloat please review and merge.

### CODE REVIEW

File: O/oneTBB/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("i686", "linux"; libc = "glibc"),
+    Platform("x86_64", "linux"; libc = "glibc"),
+    Platform("aarch64", "linux"; libc = "glibc"),
+    Platform("x86_64", "macos"; )
+]
```
Comment (giordano) [inline O/oneTBB/build_tarballs.jl:31]: Can we see if it build for more platforms?
```suggestion
platforms = supported_platforms(; experimental=true)
```

**Review by giordano — COMMENTED** [review]
Thanks for the interesting contribution!  However I think it'd be good to have it on as many platforms as possible

File: O/oneTBB/build_tarballs.jl (hunk 1 above)
Comment (pitsianis) [reply O/oneTBB/build_tarballs.jl]: The included platforms were the ones that succeeded. I have followed your suggestion, and hope that a more knowledgable in the art to customize the solutions for the platforms that fail.

**Comment by giordano** [comment]
Ok, let's go with these platforms for the time being, thanks again!

**Pull request merged**